### ISSUE TITLE
imo.im app - missed ds

### ISSUE THREAD

**Issue by evgeniy-ADG** [AdguardTeam/AdguardFilters#3041]
(https://github.com/AdguardTeam/AdguardFilters/files/482949/my_app_IMO_har.zip)

![screen shot 2016-09-20 at 18 25 11](https://cloud.githubusercontent.com/assets/15856304/18676828/981ee54c-7f5f-11e6-84f3-f4e50d7db007.png)

HTTPS filtering is on, cleared the cache but it doesn't help. We were unable to replicate it at our end.

<details>
will post here asap

**Comment by Alex-302** [comment]
@evgeniy-ADG 
У меня она в сеть не лезет, рекламы нет.
Попроси лог HTTP запросов. Только перед снятием кэш проги очистить нужно, и запомнить время запуска приложения.

**Comment by evgeniy-ADG** [comment]
@Alex-302 pls check here 1413447

**Comment by GSM789** [comment]
Dir Alex-302, pls check it...
The log file of IMO... (i Removed some unwanted part from it)...

[adguard.log.imo.modifyd.txt](https://github.com/AdguardTeam/AdguardFilters/files/485636/adguard.log.imo.modifyd.txt)


And  below, this is a new ad link that i didnt found in log file.. 


https://www.googleadservices.com/pagead/aclk?dim=330x271&nb=8&sa=L&ai=Cq5PA_DbiV5DFC9jDogPgo5ZoluK620aTvPjOjgPju_ez2gMQASDX89MhYOXK5YO0DqAB5ZrH2APIAQGpAk9MN9UmAVA-qAMByAOKBKoEvwFP0D0Jv_nr2KpstW6vITPY79PKCUu5a0m2WkE7cA8fqzwlGeWTOtZL3JLGhZ7R8mI25lE1p5calsqYW4crXnMYd4Fqegw2mO0cFjhMOGWFlNZPlumUyLXkgUR5f-3dCZTynNTVl-n6kxQ9Xz5foKx-ELjh2kTW5IZMycexEeyatDfeLYz8EtgbsQ7-b_5MY5Ukg58nBfzmMEWomxDgwd3oHgVEOmjUhSfC4sT94wF99w_xS8KlQciiZfSWmflKiYgGAZAGAdgGAoAHg-W4J5gHAagHgcYbqAemvhvYBwHSCAUIgGEQAQ&num=1&cid=CAASEuRoQkrzDZvsUFRV6_caT95x3A&sig=AOD64_37URBqVu6Rxa_Z8tYmgz7aogkVXw&client=ca-app-pub-5136333382830872&ms=CoACJg29ynmYu92FGYJWSPaAbWphvldBK3r_8kuEU9KixLQj3nSb3krzbQ-uoBTM4etFoQeErJXzLMJZr90BicszCWVZKaEDR66OmBLGxs_hd--eS-bnIbJqTK8nbDoS-kSr12tmwvyY-Ut7bN0GtNBvNzp01k4l38kKb4HmFKz-g5F1UKAe8aSlfxrrMPVIxzpog3OeXO6eJOK3sG-G4LBYvyhDzyS8hoivxRwMB67-d-TZ72DvNUuh-7QyfwSyLMy3oOxsjrOZXPKaEwOMYXqbq_8tS0RZcqx4SGWkFwOW1bHxgdvKBUNnMHOICju5TLuwy3zC8TDLjWmqf6HgoKMzGwqAArQyOHZvjeap_dqqP9C0o5Z00VDpPAQiygwc6NoNhFTz92ls


....

**Comment by Alex-302** [comment]
@GSM789 ad requests are blocked. Please try to clear cache of that app.

**Comment by GSM789** [comment]
@Alex-302 ,
 i tried all possible ways at my end... 
But still ads r visible in imo..
Any solution... 
Pls check this picture...

![_20160923_003211](https://cloud.githubusercontent.com/assets/22278693/18762149/3d153602-8125-11e6-8924-89f579ab7430.jpg)

**Comment by Alex-302** [comment]
Please check this app with disabled internet. If you are using WiFi, you can just unplug a network cable from the router.

**Comment by GSM789** [comment]
@Alex-302 , actually i didn't get that phrase...
That u mean , run imo without internet/wifi?? 
If so , then no ads..then  no use of imo without internet... 
Or u mean to disable all network connection and run imo then reconnect to internet!!??
If u dont mind, pls tell me little more detailed.... 
Thanks..

**Comment by Alex-302** [comment]
> run imo without internet/wifi?
yes.

**Comment by Alex-302** [comment]
Also you can try to enable HTTPS filtering.

**Comment by GSM789** [comment]
@Alex-302 ,
I disconnected internet connection.. cleared cache, data,force stop.., enabled Https filtering...etc.. 
Then i opened imo.. after exit, i cleared imo from my task manager..
Now i connected my phone to internet.. then run imo... No changes...
Did i miss anything....
Pls check these images..

![a](https://cloud.githubusercontent.com/assets/22278693/18763596/b2fda9ac-812b-11e6-8253-96c44c36823a.jpg)
..
![b](https://cloud.githubusercontent.com/assets/22278693/18763656/ef7e6f24-812b-11e6-862d-eed08b8d0697.png)
...
![c](https://cloud.githubusercontent.com/assets/22278693/18763698/175790de-812c-11e6-8d2a-ddc2d1ecd042.png)
..
![d](https://cloud.githubusercontent.com/assets/22278693/18763720/3192766c-812c-11e6-8584-83fa22e21e27.png)
.......

**Comment by Alex-302** [comment]
No, I mean you should to run this app without internet connection(and cleared cache).

**Comment by GSM789** [comment]
Dir alex302, Then how can i use imo for calling without internet!!!!!
I did run without internet and cache cleared...

**Comment by Alex-302** [comment]
After that please record new log of HTTP requests WITH internet connection.

**Comment by Alex-302** [comment]
> Dir alex302, Then how can i use imo for calling without internet!!!!!

ok then.
please record new log of HTTP requests WITH internet connection.

**Comment by GSM789** [comment]
Oookk.. sure.. i will..i can..

**Comment by GSM789** [comment]
NEW log file of imo.. (modified).. 

[adguardlogimonew.txt](https://github.com/AdguardTeam/AdguardFilters/files/488772/adguardlogimonew.txt)

**Comment by Alex-302** [comment]
I don't see advertising requests, which are not blocked.

**Comment by Alex-302** [comment]
Which filtering mode are you using - VPN or proxy(local or manual)?

**Comment by GSM789** [comment]
dir Alex-302,
Im using LOCAL -VPN... 
but i didnt install/enable the certificate for advanced https filtering.... 
This is my favourite method...
For blocking these ads from imo,  in which easylist u added the rules?????(default english easylist or any other)!!

**Comment by Alex-302** [comment]
@GSM789 HTTPS filtering is not working without installed certificate.

**Comment by GSM789** [comment]
But without this other vpn adblocker working good... 
I dint installed the certificate bcz of a kind of security issues...
If i installed it, then im afraid to do banking... 
Are u  exactly sure about its privacy!!!!!! 




![screenshot_2016-09-24-22-56-51](https://cloud.githubusercontent.com/assets/22278693/18810100/237dc39e-82ab-11e6-860a-a6c74ff9560f.png)

**Comment by Alex-302** [comment]
We can add your banking to HTTPS exclusions. What are you using - an app or a web version?

**Comment by GSM789** [comment]
Dir alex-302,
  for banking im using both methods.. an app and web...
Bcz Some transactions need web and some need app..
...
I ll try/check imo with installed certificate.. then i ll msg to u... (I think imo ads aren't https)..

**Comment by GSM789** [comment]
Dir alex-302, 
Still ads with https... 

![screenshot_2016-09-27-14-14-12](https://cloud.githubusercontent.com/assets/22278693/18866955/beb0f14a-84bf-11e6-89fd-9bb79e2d3bcc.png)

![screenshot_2016-09-27-14-29-18](https://cloud.githubusercontent.com/assets/22278693/18866966/d805c094-84bf-11e6-873c-d652172746ce.png)


![screenshot_2016-09-27-14-29-48](https://cloud.githubusercontent.com/assets/22278693/18866980/e6292116-84bf-11e6-98cc-ea82ff8d7548.png)


....

There are few problems After installing the https certificate,  bcz network not connecting properly to the internet... To connect internet, its taking more time and need to do retry several times in browser.. And also after few minutes automatically dropping the internet connection for an unknown reason...!!!!!! (Adguard running always at background and no restrictions)....


![screenshot_2016-09-27-14-20-47](https://cloud.githubusercontent.com/assets/22278693/18867291/4796c9c0-84c1-11e6-8068-b5c8d62f3992.png)

.....
What u say!!!

**Comment by Alex-302** [comment]
Please stop the app with ads, clear cache, then run it. When banner will be loaded, stop Adguard protection and send me log, and add time when you run that app.
Than you can disable HTTPS filtering(you should to clear credentials in Android setings->Security )

**Comment by GSM789** [comment]
Dir alex302,
I did it already..
"""" Please stop the app with ads, clear cache, then run it. When banner will be loaded, stop Adguard protection and send me log, and add time when you run that app.
Than you can disable HTTPS filtering(you should to clear credentials in Android setings->Security ) """"

So i will send you the log file soon... I need some time to remove unwanted parts from it...

But why getting network problem after https certificate installed!!!! (Now i cleared credentials bcz of this problem)..

**Comment by GSM789** [comment]
Did you do anything for secure banking with https filtering!!?????

**Comment by GSM789** [comment]
...

[imolognewhttps.txt](https://github.com/AdguardTeam/AdguardFilters/files/496259/imolognewhttps.txt)


...

**Comment by Alex-302** [comment]
All advertising requests in the log  are blocked. I don't know why it is not blocked on your device,
Please create a new issue here:
https://github.com/AdguardTeam/AdguardForAndroid

**Comment by GSM789** [comment]
Soo saaad.... 
I ll check it once again...
Anyway thanks alex-302..

**Comment by GSM789** [comment]
New issue created...
Check it ::

https://github.com/AdguardTeam/AdguardForAndroid/issues/816

**Comment by Alex-302** [comment]
Thank you)
